src/drivers/wayland/lv_wayland.c

```diff
@@ -336,6 +404,16 @@ static void handle_global(void * data, struct wl_registry * registry, uint32_t n
     LV_UNUSED(version);
     LV_UNUSED(data);
 
+    if(strcmp(interface, "wl_output") == 0) {
```

Comment:
```suggestion
    if(strcmp(interface, wl_output_interface.name) == 0) {
```

Can't test but should be ok? And follows the rest of the code convention better

Reply:
wl_output_interface is part of wayland-client-protocol.h but wl_output_interface.name is not. So using a string is the only option.